tests/performance-test/deploy/qdrouterd.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: interconnectedcloud.github.io/v1alpha1
+kind: Qdr
+metadata:
+  name: qdr-test
+  namespace: sa-telemetry
+spec:
+  deploymentPlan:
+      image: docker-registry.default.svc:5000/sa-telemetry/amq-interconnect:1.4-7
+      role: edge
+      size: 1
+      placement: Any
+  addresses:
+    - prefix: closest
+      distribution: closest
+    - prefix: multicast
+      distribution: multicast
+    - prefix: unicast
+      distribution: closest
+    - prefix: exclusive
+      distribution: closest
+    - prefix: broadcast
+      distribution: multicast
+    - prefix: collectd
```

Comment:
wouldn't this be collectd/telemetry?

Reply:
This I directly took from your config for the SAF router, so I figured I shouldn't change it?

Reply:
Oh really? Hrmmm... something seems off here then. I think Chris updated this to `collectd/telemetry` and `collectd/notify` for the metrics vs events addresses.

For testing purposes, whatever works I suppose, but I would have expected this to be `telemetry` not just `collectd`. I could be misunderstanding though.

Reply:
It works either way so I will go ahead and change it so we don't get unintended messages skewing our data later on